templates/extras/calc.md

```diff
@@ -1,49 +1,27 @@
-<div class="alert alert-info">
-<p>
-We are currently updating the Lean community website to describe working with Lean 4,
-but most of the information you will find here today still describes Lean 3.
-</p>
-<p>
-Pull requests updating this page for Lean 4 are very welcome.
-There is a link at the bottom of this page.
-</p>
-<p>
-Please visit <a href="https://leanprover.zulipchat.com">the leanprover zulip</a>
-and ask for whatever help you need during this transitional period!
-</p>
-<p>
-The website for Lean 3 has been <a href="https://leanprover-community.github.io/lean3/">archived</a>.
-If you need to link to Lean 3 specific resources please link there.
-</p>
-</div>
-
 # How to use calc
 
 `calc` is an environment -- so a "mode" like tactic mode, term mode and
 [conv mode](conv.html). Documentation and basic examples for how to use
 it are in Theorem Proving In Lean, in
-[section 4.3](https://leanprover.github.io/theorem_proving_in_lean/quantifiers_and_equality.html#calculational-proofs).
+[section 4](https://lean-lang.org/theorem_proving_in_lean4/quantifiers_and_equality.html#calculational-proofs).
 
 Basic example usage:
 
 ```lean
 example (a b c : ℕ) (H1 : a = b + 1) (H2 : b = c) : a = c + 1 :=
-calc a = b + 1 : H1
-...    = c + 1 : by rw H2
+calc a = b + 1 := H1
+     _ = c + 1 := by rw[H2]
```

Comment:
Can you adapt to putting the `calc` on its own line here and everywhere else? This is preferred style now
```suggestion
calc
  a = b + 1 := H1
  _ = c + 1 := by rw[H2]
```